azext_iot/deviceupdate/params.py

```diff
@@ -137,3 +130,302 @@ def load_deviceupdate_arguments(self, _):
+            "Supported with --by-version and no constraint flags. ",
+            arg_group="Filter",
+        )
+
+    with self.argument_context("iot device-update update import") as context:
+        context.argument(
+            "url",
+            options_list=["--url"],
+            help="Routable location from which the import manifest can be downloaded by Device Update for IoT Hub. "
+            "This is typically a read-only SAS-protected blob URL with an expiration set to at least 4 hours.",
+        )
+        context.argument(
+            "file",
+            options_list=["--file"],
+            nargs="+",
+            action="append",
+            help="Space-separated KEY=VALUE pairs corresponding to import metadata file properties. "
+            "Required key values include filename and url. --file can be used 1 or more times.",
+        )
+        context.argument(
+            "hashes",
+            options_list=["--hashes"],
+            nargs="+",
+            help="Space-separated KEY=VALUE pairs where the KEY is the hash algorithm used and the VALUE is the base64 encoded "
+            "update manifest file hash. At least a sha256 entry is required. "
+            "If not provided it will by calculated from the provided url.",
+        )
+        context.argument(
+            "size",
+            type=int,
+            options_list=["--size"],
+            help="File size in number of bytes. " "If not provided it will by calculated from the provided url.",
+        )
+
+    with self.argument_context("iot device-update device") as context:
+        context.argument("device_group_id", options_list=["--group-id", "--gid"], help="Device group Id.")
+        context.argument("device_class_id", options_list=["--class-id", "--cid"], help="Device class Id.")
+
+    with self.argument_context("iot device-update device list") as context:
+        context.argument(
+            "filter",
+            options_list=["--filter"],
+            help="Restricts the set of devices returned. You can filter on groupId, deviceClassId, "
+            "or groupId and deploymentStatus.",
+        )
+
+    with self.argument_context("iot device-update device health") as context:
+        context.argument(
+            "filter",
+            options_list=["--filter"],
+            help="Device health filter. Supports the properties of state or deviceId. "
+            "If deviceId is provided moduleId can be optionally specified.",
+        )
+
+    with self.argument_context("iot device-update device class") as context:
+        context.argument(
+            "installable_updates",
+            options_list=["--installable-updates"],
+            help="Flag indicating the command should fetch installable updates for the device class.",
+            arg_group="Update",
+            arg_type=get_three_state_flag(),
+        )
+        context.argument(
+            "best_update",
```

Comment:
Is **best-update** same as latest supported update for the device? It would be good to elaborate on the meaning.